### DISCUSSION TITLE
ReferenceError: require is not defined in ES module scope, you can use import instead

### DISCUSSION THREAD

**Discussion by thewebalchemist** [vercel/vercel#8347] [Help] [answered]
I am getting the following error while deploying my Nextjs app.

This file is being treated as an ES module because it has a '.js' file extension and '/vercel/path0/node_modules/convertkit-react/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.

**Comment by swarnava** [comment, marked as answer]
You probably have "type": "module" in your package.json file. If you remove that it will fix this.
